starter-project/src/app/api/user/accounts/route.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextResponse } from 'next/server';
+import { Bruinen } from '@bruinenco/bruinen-sdk';
+const _ = require('lodash');
```

Comment:
`import * as _ from lodash`

Reply:
@tevonsb I get this message from VSCode when I try to use this syntax:

Could not find a declaration file for module 'lodash'. 'lodash.js' implicitly has an 'any' type.
Try `npm i --save-dev @types/lodash` if it exists or add a new declaration (.d.ts) file containing `declare module 'lodash';`

Reply:
Lmk if either of those options are things I should do — wanted to avoid adding extra setup steps or files to the project if possible